ui/app.js

```diff
@@ -4370,7 +4370,7 @@ $(document).ready(() => {
     } else if (eventData.type == "call") {
       const value = eventData.data;
       DispatchMAP(value);
-      if (value && value.job.includes(playerJob)) {
+      if (value && (value.job.includes(playerJob) || value.job.includes("FirstResponder"))) {

```

Comment:
This seems like a very specific change, that might be better suited to add into the config or on your specific fork, rather than added into the framework. The config has sections for Config.PoliceJobs,  Config.AmbulanceJobs, Config.DojJobs

I'm curious how many folks actually have a job "FirstResponder" or if this is more so a specific job type you have on your server? IE: Most servers have "police" and "ambulance"

We should aim to support the main job types in this framework rather than add specific job types that may not be used by the majority of consumers.

Reply:
This isn't something that I built custom for my own fork/server.  The reason why I submitted this pull request is because ps-dispatch has two alerts that, by default, are configured to only send "FirstResponder" as the job.  See below:

[firstresponder](https://github.com/Project-Sloth/ps-dispatch/blob/e31944a7059ddaf4e95722885dd5a6a252c51898/client/cl_events.lua#L549)
https://github.com/Project-Sloth/ps-dispatch/blob/e31944a7059ddaf4e95722885dd5a6a252c51898/client/cl_events.lua#L581

This means that, out of the box, those two dispatch alerts will not show up in the call list of any player unless they either make the same modification that I do or add additional jobs to the data that gets passed, which some people may not realize they need to do.  Another solution is to edit the ps-dispatch functions to pass additional jobs, but that wouldn't solve the issue of a user wanting to pass just a single "FirstResponder" value.

Reply:
Both an oversight on there, we will accept a PR correcting that so it sends to all jobs

Reply:
> @DonHulieo was this part of your changes by chance? I doubt it

Nah mate, I haven’t made any PR’s to MDT. I haven’t implemented my method for checking police jobs in the MDT yet, and calling FirstResponder like that (because as you said it’s not a ‘job’) will just return nil in a traditional job check without my additional logic.

Reply:
> > @DonHulieo was this part of your changes by chance? I doubt it
> 
> Nah mate, I haven’t made any PR’s to MDT. I haven’t implemented my method for checking police jobs in the MDT yet, and calling FirstResponder like that (because as you said it’s not a ‘job’) will just return nil in a traditional job check without my additional logic.

Yeah, I figured as much. That was part of the ps-dispatch, since you have been working on the dispatch, you think you can work on this for the dispatch? We'll then go ahead and adjust things on ps-mdt

Reply:
> figured as much. That was part of the ps-dispatch, since you have been working

If I have edit permission for this PR, I can see if I can implement my changes on here, or should I just create my own PR?

Reply:
@phillyCHEEEEEZ Thanks for the details, wasn't aware that was part of dispatch. Looks like we've surfaced the issue tho and got the right conversation going.

Reply:
> 

Feel free to send your own PR and we will get @phillyCHEEEEEZ to edit this one as well so they can get credit for this 

Thanks everyone